x-pack/plugins/cloud_integrations/cloud_data_migration/public/application/components/app.tsx

```diff
@@ -0,0 +1,146 @@
+          <EuiImage
+            alt={i18n.translate('xpack.cloudDataMigration.illustration.alt.text', {
+              defaultMessage: 'Illustration for cloud data migration',
+            })}
+            css={styles.illustrationCss}
+            src={`${basePath}/plugins/kibanaReact/assets/` + 'illustration_cloud_migration.png'}
+          />
+        </EuiFlexItem>
+        <EuiFlexItem>
+          <EuiTitle size="s">
+            <h1>
+              <FormattedMessage
+                id="xpack.cloudDataMigration.migrateToCloudTitle"
+                defaultMessage="Elastic Cloud helps your data work harder."
+              />
+            </h1>
+          </EuiTitle>
+
+          <EuiSpacer size="xl" />
+
+          <EuiListGroup maxWidth={700}>
+            <EuiListGroupItem
+              {...listItemProps}
+              label={
+                <EuiText css={styles.listItemCss} size="s" color="subdued">
+                  <FormattedMessage
+                    id="xpack.cloudDataMigration.deployInSeconds.text"
+                    defaultMessage="Deploy Elastic quickly and scale up to petabytes of data with a click."
+                  />
+                </EuiText>
+              }
+            />
+
+            <EuiListGroupItem
+              {...listItemProps}
+              label={
+                <EuiText css={styles.listItemCss} size="s" color="subdued">
+                  <FormattedMessage
+                    id="xpack.cloudDataMigration.freeUpEngineering.text"
+                    defaultMessage="Streamline your Elastic workflow and free up your team. Manage multiple deployments from a single view and centralize your monitoring data."
+                  />
+                </EuiText>
+              }
+            />
+
+            <EuiListGroupItem
+              {...listItemProps}
+              label={
+                <EuiText css={styles.listItemCss} size="s" color="subdued">
+                  <FormattedMessage
+                    id="xpack.cloudDataMigration.getHelpFromCreators.text"
+                    defaultMessage="Get support from the creators of Elastic to help you ingest all sorts of data and tune your performance."
+                  />
+                </EuiText>
+              }
+            />
+
+            <EuiListGroupItem
+              {...listItemProps}
+              label={
+                <EuiText css={styles.listItemCss} size="s" color="subdued">
+                  <FormattedMessage
+                    id="xpack.cloudDataMigration.getInstantAccess.text"
+                    defaultMessage="Access the latest version of Elastic with features you want, like anomaly detection, searchable snapshots, advanced security, and so much more."
+                  />
+                </EuiText>
+              }
+            />
+          </EuiListGroup>
+
+          <EuiSpacer size="l" />
+
+          <div>
+            <EuiButton fill={true} target="_blank" href="https://ela.st/cloud-migration">
```

Comment:
We'll want to track telemetry here; opened https://github.com/elastic/kibana/issues/147474 to follow up.